aws/table_aws_s3_bucket.go

```diff
@@ -145,7 +147,7 @@ func tableAwsS3Bucket(_ context.Context) *plugin.Table {
 				Description: "A container for specifying the notification configuration of the bucket. If this element is empty, notifications are turned off for the bucket.",
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getS3BucketEventNotificationConfigurations,
-				Transform:   transform.FromValue(),
+				Transform:   transform.FromValue().Transform(removeMedataFromS3BucketEventNotificationConfigurations),
```

Comment:
```suggestion
				Transform:   transform.FromValue().Transform(removeMetadataFromS3BucketEventNotificationConfigurations),
```

Reply:
Can you please search for any instances of misspelled `metadata` words, e.g., `Medata`?